packages/lesswrong/lib/ultraFeed.ts

```diff
@@ -14,6 +14,11 @@ export const FeedCommentThreadFragment = gql(`
   fragment FeedCommentThreadFragment on FeedCommentThread {
     _id
     commentMetaInfos
+    isOnReadPost
```

Comment:
does `PostsListWithVotes` not contain `isRead` (assuming that's checking the same thing)?